server/core/src/main/java/dev/slimevr/filtering/QuaternionMovingAverage.kt

```diff
@@ -62,13 +65,13 @@ class QuaternionMovingAverage(
 
 				// Slerps the target rotation to that predicted rotation by
 				// a certain factor.
-				filteredQuaternion = filteredQuaternion.interpR(quatBuf, predictFactor * fpsTimer.timePerFrame)
+				filteredQuaternion = filteredQuaternion.interpR(quatBuf, predictFactor * (fpsTimer?.timePerFrame ?: 1f))
```

Comment:
I really don't see a case where fpsTimer should be null... idk I guess this is fine, but it's an edge case that maybe should be detected and reported instead of potentially introducing unexplained weird behaviour?

Reply:
It’s because this code can potentially run before vrserver lateinit

Reply:
Hmm alright.